### PR TITLE
[docs] Update wrong file extension used in mv command

### LINKED ISSUE #2582: [Docs] The extension of the docker-compose file in deployment/docker documentation step 4 is wrong
### Version Information
N/A
### Environment

Docker (Documentation)

### What is the expected behaviour?

A command to rename a "curl"ed Dockerfile is given in step 4 of the documentation, as shown below. The command should execute without any errors.
```
curl -LO https://raw.githubusercontent.com/ToolJet/ToolJet/main/deploy/docker/docker-compose-postgres.yaml
mv docker-compose-postgres.yml docker-compose.yml
```

### What is the current behaviour?

An error: No such file or directory. 
```
mv: cannot stat 'docker-compose-postgres.yml': No such file or directory
```

### How to reproduce the issue?

1. Follow [this](https://docs.tooljet.com/docs/deployment/docker/) documentation
2. In step 4, confirm the bug

### Screenshots or Screencast

image

### Please provide any traces or logs that could help here.
```
mv: cannot stat 'docker-compose-postgres.yml': No such file or directory
```

### Any possible solutions?
Change the extension of the file in the `mv` command from `.yml` to `.yaml`

### Can you identify the location in the driver source code where the problem exists?
N/A

### If the bug is confirmed, would you be willing to submit a PR?
Yes

### PR DESCRIPTION
**Pull request by MithunKinarullathil**
- Fixes #2582

### CODE REVIEW

File: docs/docs/deployment/docker.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ OR
 For PostgreSQL database setup with persistent volume:
 ```bash
 curl -LO https://raw.githubusercontent.com/ToolJet/ToolJet/main/deploy/docker/docker-compose-postgres.yaml
```
Comment (akshaysasidrn) [inline docs/docs/deployment/docker.md:41]: @MithunKinarullathil Could you rename the corresponding file name as well? Currently its `docker-compose-postgres.yml`

**Review by akshaysasidrn — CHANGES_REQUESTED** [review]

File: docs/docs/deployment/docker.md (hunk 1 above)
Comment (MithunKinarullathil) [reply docs/docs/deployment/docker.md:41]: Done. Although I think it should be the other way around, as in changing everything in README to yml, just not to break anything for existing users?

Comment (akshaysasidrn) [reply docs/docs/deployment/docker.md:41]: That is alright, as readme was already incorrect.

**Review by akshaysasidrn — APPROVED** [review]

**Comment by akshaysasidrn** [comment]
Thank you for the contribution!

File: docs/docs/deployment/docker.md (hunk 1 above)
Comment (MithunKinarullathil) [reply docs/docs/deployment/docker.md:41]: Cool.

**Pull request merged**